README.md

```diff
@@ -650,6 +650,9 @@ While the `schema` is currently validated for any developer errors,
 there is no guarantee that supplying user-generated schema could not
 expose your application to remote attacks.
 
+The _data_ that `fast-json-stringify` serializes is trusted too. In other terms,
+fast-json-stringify makes no claim to validate the incoming data against the schema. 
```

Comment:
```
The _data_ that `fast-json-stringify` serializes should be a trusted source. 
The user should take their responsibility to sanitize the _data_ before passing 
to `fast-json-stringify`. The validation in this package is only against the `schema` 
but not `malicious code`.
```

Reply:
Would it be better? @ivan-tymoshenko

Reply:
That would work for me.

Reply:
What do you mean by `malicious code`? Whatever it means I agree with this point, because FJS doesn't check your data for any `malicious code`. But I think you talking about something different. Sanitizing is not validation.

Reply:
Ok, I saw the original security report. I would say that: fast-json-stringify guarantees that you will get a valid output only if your input matches the schema or can be coerced to the schema. If your input doesn't match the schema, you will get undefined behavior.

Reply:
See https://github.com/fastify/fast-json-stringify/issues/559 for more context

Reply:
PTAL